src/components/Navigator.vue

```diff
@@ -204,7 +206,18 @@ export default {
 
 .loading-placeholder {
   color: var(--color-figure-gray-secondary);
-  padding: 12px;
-  @include font-styles(body-reduced);
+  height: 100%;
+  padding: $section-spacing-single-side 0;
+  text-align: center;
+  width: 100%;
+}
+
+.loading-spinner {
+  --spinner-size: 40px;
+
+  height: var(--spinner-size);
+  position: sticky;
```

Comment:
Why using position sticky instead of just centering with flex? Is there any use case that I'm missing?

Reply:
I was trying to think of a simpler approach like that at first, but I couldn't think about how to make it work right since it needs to vertically center in the sidebar, which may expand beyond the viewport. I'll try to see if I can come up with something simpler like that and make it work.

Reply:
Yeah, I'm not sure how to do this with a flex layout. The trouble happens when you scroll down the page, we want to still show the loading indicator in the center of the sidebar instead of having it scroll out of view.

Let me know if you think of a better way of doing that though please!

Reply:
Alternatively, you could do:

```css
.loading-placeholder {
  color: var(--color-figure-gray-secondary);
  position: sticky;
  top: $nav-height;
  height: calc(100vh - #{$nav-height});
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;

  @include breakpoint(medium, nav) {
    position: static;
    height: 100%;
  }
}

.loading-spinner {
  --spinner-size: 40px;

  height: var(--spinner-size);
  width: var(--spinner-size);
}
```

just as we do for class `.navigator-card-inner`
and extract the common CSS into a SCSS mixin.

It implies to use position sticky but at least you don't have to center the position with `top`
As you prefer :)

I'm approving the PR.